impeller/tessellator/tessellator_unittests.cc

```diff
@@ -82,6 +83,52 @@ TEST(TessellatorTest, TessellatorBuilderReturnsCorrectResultStatus) {
     ASSERT_EQ(polyline.points.size(), 2u);
     ASSERT_EQ(result, Tessellator::Result::kInputError);
   }
+
+  // More than 30 contours, non-zero fill mode.
+  {
+    Tessellator t;
+    PathBuilder builder = {};
+    for (auto i = 0; i < 60; i++) {
```

Comment:
Instead of using a hardcoded 60, you can use the variable you created now to make this test a bit more robust to changes in that number.

Reply:
Done